src/containers/guest-home-page/WhoWeAre.js

```diff
@@ -0,0 +1,53 @@
+import { useTranslation } from 'react-i18next'
+import { Box } from '@mui/material'
+
+import TitleWithDescription from '~/components/title-with-description/TitleWithDescription'
+import videoImg from '~/assets/img/guest-home-page/videoImg.svg'
+
+const titleVariant = {
+  md: 'h3',
+  xs: 'h4'
+}
+
+const descriptionVariant = {
+  xs: 'subtitle1'
+}
+
+const style = {
+  container: {
+    marginBottom: '70px',
+  },
+  videoWrapper : {
+    px: '24px'
+  },
+  video: {
+    display: 'block',
+    margin: '0 auto',
+    maxWidth: '100%',
+  }
+}
+
+
+const WhoWeAre = () => {
+  const { t } = useTranslation()
+
+  return (
+    <Box sx={ style.container }>
```

Comment:
use className 'section'. It is global class with marginBottom placed in index.css

Reply:
Fixed.